src/components/Pagination/Pagination.tsx

```diff
@@ -2,0 +1,80 @@
+import classNames from 'classnames';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange?: (page: number) => void;
+};
+
+export const Pagination = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange = () => {},
+}: Props) => {
+  const lastPage = Math.ceil(total / perPage);
+
+  return (
+    <ul className="pagination">
+      <li
+        className={classNames('page-item', {
+          disabled: currentPage === 1,
+        })}
+      >
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          onClick={() => {
+            if (currentPage > 1) {
+              onPageChange(currentPage - 1);
+            }
+          }}
+          aria-disabled={currentPage === 1 ? 'true' : 'false'}
```

Comment:
It's a good practice to use button elements for actions that do not lead to a navigation event. Since the prev link is used to change the page in a pagination component and not to navigate to a different URL, it would be more semantically correct to use a button element here.